src/modules/launcher/PowerLauncher/ReportWindow.xaml.cs

```diff
@@ -50,8 +61,7 @@ private static Paragraph Hyperlink(string textBeforeUrl, string url)
             var link = new Hyperlink { IsEnabled = true };
             link.Inlines.Add(url);
             link.NavigateUri = new Uri(url);
-            link.RequestNavigate += (s, e) => Process.Start(e.Uri.ToString());
-            link.Click += (s, e) => Process.Start(url);
```

Comment:
[clarification] What does `link.Click` do and was this functionality removed ?

Reply:
I removed it because both the `RequestNavigate` as well as `Click` event were trying to open the same URL. When we register for both the events, on clicking the hyperlink there were two events being fired, leading to two issue pages opening up. That's why I removed one of them. 

Also, I removed the onClick event and retained the RequestNavigate event because the requestNavigateEventHandler has information about the redirecting url whereas the RoutedEventArgs (of the Click event handler) don't and we would have to pass it as an argument/refer to some hard coded string.